setup.cfg

```diff
@@ -1,23 +0,0 @@
-[tool:pytest]
```

Comment:
Aren't we using this?

Reply:
I think it's outdated. We can remove both this and setup.py whenever / if we move to pyproject.toml